vochain/transaction/account_tx.go

```diff
@@ -239,3 +240,133 @@ func (t *TransactionHandler) SetAccountInfoTxCheck(vtx *vochaintx.Tx) error {
 	}
 	return nil
 }
+
+// DelSikTxCheck checks if a delete sik tx is valid
+func (t *TransactionHandler) DelSikTxCheck(vtx *vochaintx.Tx) (common.Address, error) {
```

Comment:
Shouldn't we check if the SIK can actually be deleted?  So it must exist already and not be invalidated.